src/cephadm/cephadmlib/daemons/nvmeof.py

```diff
@@ -111,6 +111,7 @@ def customize_container_mounts(
             )
         else:
```

Comment:
@adk3798.do we still need the mtls directory logic? It looks like the `_get_tls_cert_key_mounts()` function maps keys to the root (/) directory. What do you think?

Reply:
I have absolutely no idea what that directory is for so I didn't touch it. Since I know this patch works I'm going to merge it. If you think we don't need that directory now, feel free to open a PR yourself to change that.